module/VuFind/src/VuFind/ILS/Driver/AbstractBase.php

```diff
@@ -81,4 +81,28 @@ protected function throwAsIlsException(
     ): void {
         throw new ILSException($msg ?? $exception->getMessage(), 0, $exception);
     }
+
+    /**
+     * Explode a delimited setting to an array
+     *
+     * @param string $value     Setting value
+     * @param bool   $trim      Whether to trim the values
+     * @param string $separator Separator
+     *
+     * @return array
+     */
+    protected function explodeSetting(
+        string $value,
+        $trim = false,
```

Comment:
Is there a compelling reason NOT to trim by default? I can't think of a scenario where we would want to keep random whitespace, and I suspect that the only reason the code doesn't already trim everywhere is laziness/inconsistency.

Reply:
I considered this and decided to err on the side of caution because particularly with codes from an ILS, a single space could be a valid one. We could of course try to do something smart to allow an intentional space while trimming everything else, but I'm not sure if that's worth the complexity.

Another point of view is that this code will potentially be executed millions of times, while you only have to get the config right once. :)

Reply:
All good points. I won't argue against them. :-)